### PR TITLE
Remove mailgun (email) from the opt-out

### PR DESCRIPTION
**Pull request by commuterjoy**
Noisy - https://app.getsentry.com/nextftcom/ft-next-opt-in-out/group/91678287/

We use Slack + Google Spreadsheets instead.

/cc @keirog look ok?

### CODE REVIEW

File: server/lib/feedback/send-email.js [hunk 1]
```diff
@@ -1,78 +0,0 @@
-/*jshint node:true*/
-'use strict';
-
-const util = require('util');
-const mailgun = require('mailgun-js')({ apiKey: process.env.MAILGUN_API_KEY, domain: process.env.MAILGUN_DOMAIN });
-const Slack = require('node-slack');
```
Comment (keirog) [inline server/lib/feedback/send-email.js]: We want to keep the Slack bits!

Comment (commuterjoy) [inline server/lib/feedback/send-email.js]: Hah. Oops - I just saw a file called 'send-email' and deleted it :)

**Comment by keirog** [comment]
:+1:

**Pull request merged**